tools/rgrnastar/rg_rnaStar.xml

```diff
@@ -680,6 +668,40 @@ used: >=5 mappings => MAPQ=0; 3-4 mappings => MAPQ=1; 2 mappings => MAPQ=3. This
             <output name="mapped_reads" file="rnastar_test_mapped_reads.bam" compare="sim_size" delta="634" />
             <output name="transcriptome_mapped_reads" file="rnastar_test_transcriptome_mapped_reads.bam" compare="sim_size" delta="634" />
         </test>
+        <!-- test cached no index but gtf file and GeneCounts TranscriptomeSAM mode -->
+        <test expect_num_outputs="5">
+            <conditional name="singlePaired">
+                <param name="sPaired" value="single" />
+                <param name="input1" value="tophat_in2.fastqsanger" ftype="fastqsanger" />
+            </conditional>
+            <conditional name="refGenomeSource">
+                <param name="geneSource" value="indexed" />
+                <conditional name="GTFconditional">
+                    <param name="GTFselect" value="without-gtf-with-gtf" />
```

Comment:
Shouldn't the value be either `without-gtf` or `with-gtf`?

Really wondering why the test does not fail .. If I'm right.

Reply:
See above.